bindings/c/.clang-format

```diff
@@ -0,0 +1,250 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
```

Comment:
Can we choose one of the built-in formats? It may not be necessary to define our own clang-format file.

Reply:
The clang-format in the PR is the built-in formats of Google.

Reply:
so, why not `--style=Google`

Reply:
> so, why not `--style=Google`

the .clang-format file is generated by
```
clang-format --style=Google -dump-config > .clang-format
```

I am not sure what you mean, but I guess you want to pass -style=Google directly in makefile?

Reply:
> but I guess you want to pass -style=Google directly in makefile?

Yes, I think we don’t need to maintain a file that won’t change. We just need to mark that we use the Google style.